packages/amplify_authenticator/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 0.1.0-rc.3 (2022-01-12)
```

Comment:
Shouldn't we have something in the changelog here?

Reply:
The only change is that the authenticator not requires a higher version of amplify_flutter.

Reply:
Updated